### ISSUE TITLE
Combine semantic relation and contextual proximity relation dataframes

### ISSUE THREAD

**Issue by kuoirene** [amosproj/amos2024ss05-knowledge-graph-extractor#31]
## User story
1. As a developer
2. I need to combine the semantic relation and contextual proximity relation dataframes into one
3. So that I have a network graph dataframe that is a graph of concepts for our text

## Acceptance criteria
* One dataframe that connects each node concept with other node concepts with all respective edges (relations)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
